draft-yasskin-http-origin-signed-responses.md

```diff
@@ -794,15 +777,14 @@ Accept-Signature: mi/mi-sha256
 states that the client will accept signatures with payload integrity assured by
 the `MI` header and `mi-sha256` content encoding and implies that the client
 will accept integrity assured by the `Digest: SHA-256` header and signatures
-from 2048-bit RSA keys and ECDSA keys on the secp256r1 and secp384r1 curves.
+from ECDSA keys on the secp256r1 curve.
 
 ~~~http
-Accept-Signature: -rsa/2048, rsa/4096
+Accept-Signature: -ecdsa/secp256r1, ecdsa/secp384r1
```

Comment:
Nit: You don't want the ecdsa/secp384r1 here, right?

Reply:
It was an intentional part of the example, showing how an odd client, that changed which curve is supported, would announce that fact to servers. Let me know if you think I'm wrong to have an example like that.

(I'm not certain my `Accept-Signature` thing is a good idea overall, but that's a discussion for another PR.)

Reply:
Oh! Sorry, I got confused and read the text above as describing this example rather than the text below. Ignore me.